spec/gameSpec.js

```diff
@@ -0,0 +1,47 @@
+describe  ('Game', function() {
+  var bowling;
+
+  beforeEach(function() {
+    bowling = new Game();
+  })
+
+  it('can score a gutter game', function() {
+    for(let i = 0; i < 20; i++) { bowling.roll(0) }
+    expect(bowling.getScore()).toEqual(0)
+  })
+
+  it('can score 20 points', function() {
+    for(let i = 0; i < 20; i++) { bowling.roll(1) }
+    expect(bowling.getScore()).toEqual(20)
+  })
+
+  it('can roll multiple balls and succesfully accumulate points', function() {
+    bowling.roll(4)
+    bowling.roll(3)
+    for(let i = 0; i < 18; i++) { bowling.roll(0) }
+    expect(bowling.getScore()).toEqual(7)
+  })
+
+  it('can earn bonus in case of spare', function() {
+    bowling.roll(5)
+    bowling.roll(5)
+    bowling.roll(5)
+    for(let i = 0; i < 18; i++) { bowling.roll(0) }
+    expect(bowling.getScore()).toEqual(20)
```

Comment:
I love seeing the logic as I go down these tests as you add each new feature. Shows you followed a good TDD process of a red, green, refactor cycle